### PR TITLE
Add NAMD to documentation

### PR DESCRIPTION
**Pull request by RMeli**
* Add NAMD to documentation, following #85
* Use official names for packages in index

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -2,11 +2,12 @@ site_name: alps-uenv
 nav:
   - Home: index.md
   - 'uenv':
-    - 'gromacs': uenv-gromacs.md
-    - 'quantum-espresso': uenv-qe.md
-    - 'cp2k': uenv-cp2k.md
-    - 'linaro-forge': uenv-linaro-forge.md
-  - 'packaging':
+    - 'CP2K': uenv-cp2k.md
```
Comment (bcumming) [inline mkdocs.yml:5]: the names matched the names of the uenv on the systems, i.e the name that is displayed with `uenv image find` and used for `uenv start`, `srun --uenv=` etc.

How about having both the full name and the uenv name in the header (e.g. 'Linaro Forge: `linaro-forge`')?

Comment (RMeli) [reply mkdocs.yml:5]: I see. I thought it was a bit ugly to have the `uenv` name in the menu, but what bothered me more was the ordering. I can revert the name if preferred, but your suggestion sounds good too.

Comment (bcumming) [reply mkdocs.yml:5]: Let's use my suggestion, which will hopefully make the page more searchable.

**Review by bcumming — APPROVED** [review]

**Comment by RMeli** [comment]
Thanks @bcumming, I forgot to push the update.

**Pull request merged**